implemented for pip wheels

longyuxi opened the pull request:
Hi,

Thanks for the great resource. I implemented this for pip wheels and included the table for PyTorch 2.7.0-2.8.0.

Note that currently my code searches for 'manylinux_2_28_x86_64' in the wheel name, which is only done for releases 2.7.0 and beyond. I suspect there is some packaging change between 2.6 and 2.7, but I don't think it is of much value to painstakingly implement proper parser for all previous releases since from reading the existing `table.md` and my `table_pip.md`, architectures supported by 2.5.1 from conda are the same as architectures supported by 2.7.0 pip, so we can assume the same architectures are supported by 2.5.1-2.7.0 pip.

longyuxi committed:
implemented for pip wheels

moi90 commented on pytorch_compute_capabilities_pip.py:
```diff
@@ -0,0 +1,623 @@
+"""
+Analyze all PyTorch 2.x versions and generate comprehensive table. Note the inclusion of'manylinux_2_28_x86_64' is release name will only find 2.7.0 and newer (currently through 2.8.0).
```
@longyuxi Could you use the explanation from the pull request description here?

moi90 commented on pytorch_compute_capabilities_pip.py:
```diff
@@ -0,0 +1,623 @@
+    try:
+        cuobjdump_cmd = "cuobjdump"
+        # I'm running this on a compute cluster with a singularity container
+        # cuobjdump_cmd = "singularity exec --bind /gpfs/mskmind_ess/longy/ /gpfs/mskmind_ess/longy/singularity/cuda.sif cuobjdump"
```
@longyuxi I would just remove this comment.

moi90 commented:
Thanks for your valuable contribution! Could you address my and Copilot's above comments?

longyuxi committed:
add pytorch 2.9.0 and formatting fixes

longyuxi commented:
Glad you like it! I have implemented the fixes you requested and fixed one other minor formatting issue (the table now reports the pip wheel name as the package name, instead of formatting to align with the conda package name).

Also, coincidentally PyTorch 2.9.0 was released a few days ago so the updated table includes that too.
